common/app/views/support/GetClasses.scala

```diff
@@ -17,6 +17,8 @@ object GetClasses {
     RenderClasses(Map(
       ("fc-item", true),
       ("js-fc-item", true),
+      ("fc-item--" + item.pillar.name.toLowerCase(), true),
```

Comment:
Should these only be added if the switch is true? Otherwise these classes don't really mean anything.

Reply:
yeah good thinking